Lower the minimum Flask version

Conan-Kudo opened the pull request:
This module works fine with Flask 0.12 and newer, so lowering the
version required makes it possible to run with Flask shipped in
RHEL/CentOS 8 and other LTS distributions.

Signed-off-by: Neal Gompa <ngompa13@gmail.com>

abompard commented:
Actually I'd like to keep poetry's semver feature that will not allow upgrades to a backwards incompatible Flask version, could you use `[">=0.12", "<2.0.0"]` for the version specification please?

Conan-Kudo committed:
Lower the minimum Flask version

This module works fine with Flask 0.12 and newer, so lowering the
version required makes it possible to run with Flask shipped in
RHEL/CentOS 8 and other LTS distributions.

Signed-off-by: Neal Gompa <ngompa13@gmail.com>

Conan-Kudo commented:
@abompard Done.
